### PR TITLE
add volume count to ingest

### PR DESCRIPTION
**Pull request by dnoneill**

### CODE REVIEW

File: src/server/ingest/tasks.py [hunk 1]
```diff
@@ -578,5 +578,18 @@ def new_contributor(contrib, contrib_data):
...
+	url = '{}/portal/_search?size={}'.format(es, size) 
+	count = 0
+	data = requests.get(url).json() 
+	hits = data['hits']['hits']
+	for hit in hits:
+		identifier = hit['_source']['dublin_core']['identifier']
+		for id in identifier:
+			if 'encoding' in id and id['encoding'] == 'URI':
+				count += 1
+	print("%s volumes"%count)
```
Comment (alyxrossetti) [inline src/server/ingest/tasks.py]: change to newer string formatting syntax please :)

File: src/server/ingest/tasks.py [hunk 2]
```diff
@@ -592,5 +592,18 @@ def new_contributor(contrib, contrib_data):
     if "frequency" in contrib_data and contrib_data['frequency'] != '' and contrib_data['frequency'] != None:
         contrib.frequency = contrib_data['frequency']
     contrib.save()
-    
+   
+def volume_count(es):
+	size = settings.ELASTICSETTINGS['index']['max_result_window']
+	url = '{}/portal/_search?size={}'.format(es, size)
```
Comment (alyxrossetti) [inline src/server/ingest/tasks.py]: The url should depend on the environment.

Comment (dnoneill) [reply src/server/ingest/tasks.py]: It does. It is pulling the es url based on the process_contributions option at ingest

Comment (alyxrossetti) [reply src/server/ingest/tasks.py]: Whoops - yes it does. Apologies.

**Comment by dnoneill** [comment]
@alyxrossetti requested change merged.

**Pull request merged**